mobile/pubspec.lock

```diff
@@ -53,10 +53,10 @@ packages:
     dependency: transitive
     description:
       name: build
-      sha256: "3fbda25365741f8251b39f3917fb3c8e286a96fd068a5a242e11c2012d495777"
+      sha256: "80184af8b6cb3e5c1c4ec6d8544d27711700bc3e6d2efad04238c7b5290889f0"
```

Comment:
🧐I don't think bumping all these dependencies belongs to this PR (it's unrelated to advertised changes).

I would encourage creating small PRs that update deps in the future :)

Reply:
Happy to split this off - I thought I would finish this ticket earlier but it had a bit of a rat tail :)

Reply:
done

Reply:
I was hoping to finish this work faster.

Should have split this off properly. The initial commit is actually not complete. There were changes in the Dart code needed after upgrading the packages, but I did not separate these properly. Sorry for that. I'll pay more attention next time.